assets/installation.md

```diff
@@ -48,28 +50,27 @@ The remaining Linux instructions will provide command-line oriented instructions
 
 The `dnf` packaging tool provides support for directly installing from a URL:
 
-	```
 	# dnf install https://github.com/vegastrike/Vega-Strike-Engine-Source/releases/download/v0.6.0/vega-strike_v0.6.0-centos-8.rpm
 	# dnf install https://github.com/vegastrike/Assets-Production/releases/download/v0.6.2/vsUTCS_v0.6.2-centos-8.rpm
-	```
 
 NOTE: This doesn't work yet as we missed getting the RPMs build for the Vega Strike Engine 0.6.0 Release. See [here for details](https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/368)
 
 ### Linux: Debian/Ubuntu
 
 To install using Apt:
 
-	```
 	$ mkdir vegastrike && cd vegastrike
 	$ curl -LO https://github.com/vegastrike/Vega-Strike-Engine-Source/releases/download/v0.6.0/vega-strike_v0.6.0-Ubuntu-xenial_x86_64.deb
 	$ curl -LO https://github.com/vegastrike/Assets-Production/releases/download/v0.6.2/vsUTCS_v0.6.2.deb
-	$ sudo dpkg -i vega-strike_v0.6.0-Ubuntu-xenial_x86_64.deb
-	$ sudo dpkg -i vsUTCS_v0.6.2.deb
-	```
+	$ sudo apt install vega-strike_v0.6.0-Ubuntu-xenial_x86_64.deb
+	$ sudo apt install vsUTCS_v0.6.2.deb
```

Comment:
Looking much better. Just one more thing: I think the `./` before each filename is necessary in order for apt to recognize it as a file and install it.

Reply:
fixed

Reply:
Perfect!